src/app/configs/data/shortcuts_mac.xml

```diff
@@ -92,6 +92,51 @@
       <key>nav-escape</key>
       <seq>Esc</seq>
     </SC>
+    <SC>
+      <key>next-tab</key>
+      <seq>Ctrl+Tab</seq>
```

Comment:
On macOS, it needs to be `Meta` instead of `Ctrl`. 
For macOS, Qt namely automatically translates `Ctrl` to `Cmd`, and `Cmd`+`Tab` and `Cmd`+`Shift`+`Tab` are reserved system shortcuts. 
For in-app tab switching, most macOS apps use the actual `Ctrl` key, and to get this with Qt, you need to write `Meta`.